crates/tauri-cli/src/helpers/config.rs

```diff
@@ -172,11 +173,11 @@ fn get_internal(
     || config_path.extension() == Some(OsStr::new("json5"))
   {
     let schema: JsonValue = serde_json::from_str(include_str!("../../config.schema.json"))?;
-    let schema = jsonschema::JSONSchema::compile(&schema).unwrap();
-    let result = schema.validate(&config);
-    if let Err(errors) = result {
+    let validator = jsonschema::validator_for(&schema).expect("Invalid schema");
+    let mut errors = validator.iter_errors(&config).peekable();
+    if errors.peek().is_some() {
```

Comment:
do we really need to peek? I think we can just loop through the iterator normally and if no errors found, it will skip the iteration and nothing will happen

Reply:
Good point!

Reply:
The only thing needed is to skip `exit` in the `if !reload` block which happens only after all errors are logged if there is at least one error. Right now peek is used exactly for this, but a boolean would work too

Reply:
Sorry for the force push, I updated this block.

Reply:
> The only thing needed is to skip `exit` in the `if !reload` block which happens only after all errors are logged if there is at least one error. Right now peek is used exactly for this, but a boolean would work too

Oh didn't notice that one, apologies but you had it correct, can you revert it?

Reply:
you also need to fix Cargo.lock conflicts with `dev` branch

Reply:
Sure! reverted + updated `Cargo.lock`